packages/lit-helpers/README.md

```diff
@@ -120,3 +120,86 @@ html`
     }
   }
 </script>
+
+## Privately Settable Read-Only Properties
+
+The classic Polymer library allowed defining properties that were read-only, but privately settable. By adding `readOnly: true` to the property declaration, the property could be set with a `_setPropName` method.
```

Comment:
Not many people will understand this reference, maybe we should just document the feature as it is?